packer/install-dependencies.sh

```diff
@@ -297,8 +247,7 @@ done
 
 AZURE_CNIIMAGEBASE="mcr.microsoft.com/containernetworking"
 AZURE_CNI_NETWORKMONITOR_VERSIONS="
-0.0.6
```

Comment:
0.0.6 was there 23days ago, it means there are possible clusters with supported k8s version and still requires it for the scale up/upgrade work until we upgrade all ccps. Let's keep it for a while and then remove it.